projectaria_tools/utils/viewer_aria_sensors.py

```diff
@@ -58,7 +58,12 @@ def main():
     device_calibration = provider.get_device_calibration()
 
     # Spawn rerun and log things we want to see
-    rr.init("Aria_Sensor_Viewer", spawn=(not args.rrd_output_path))
+    rr.init("Aria Sensors Data Viewer", spawn=(not args.rrd_output_path and not args.web))
+
+    if args.web:
```

Comment:
Maybe add the same comment as the other binary
`# Run the viewer in the web browser or desktop app`

Reply:
Thanks so much for pointing it out. I would add the comment